java/client/src/org/openqa/selenium/WebDriver.java

```diff
@@ -91,11 +91,19 @@
    * found collection, or will return an empty list if the timeout is reached.
    *
    * @param by The locating mechanism to use
+   * 
    * @return A list of all {@link WebElement}s, or an empty list if nothing matches
+   * 
+   * @throws ClassCastException The WebDriver interface is generic. 
```

Comment:
Are these comments about `ClassCastException` unnecessary? `ClassCastException` is unchecked.